### DISCUSSION TITLE
Could you please make more example

### DISCUSSION THREAD

**Discussion by vit1251** [slint-ui/slint#504] [Ideas]
Could you please provide more example for example simple canvas or custom component ideas for create palette with users controls like charts and other.

**Comment by tronical** [comment]
Good ideas. I was thinking of a little example that uses for example raqote or tiny-ski for custom rendering.

Somebody recently pasted some test case in a bug report that would render a nice calendar. Perhaps that could be turned into a custom control example. Or a dial.

Do you have a preferred programming language for these?

**Comment by vit1251** [comment]
I use Rust but only in research purposes. I think that more popular today is Python or JavaScript and I already familar with this language.

Right now I review code in https://github.com/sixtyfpsui/sixtyfps/blob/master/examples/plotter/ and it may be start point for researching.

I also think that most people use HTML canvas before and it maybe good bind with technology in future.
